src/screens/Customers/CustomerProfile.tsx

```diff
@@ -94,6 +94,8 @@ function CustomerProfile(props: CustomerProps) {
     return (
       <Button
         variant="contained"
+        component={Link}
```

Comment:
I imagine this will be a problem for a lot of places. Let's do the rounding all at once so it's done in a consistent way. This is a really good point though, I opened an issue for it, thanks!